google-cloud-spanner/README.md

```diff
@@ -2,9 +2,24 @@
 
 [Google Cloud Spanner API](https://cloud.google.com/spanner/) ([docs](https://cloud.google.com/spanner/docs)) provides a fully managed, mission-critical, relational database service that offers transactional consistency at global scale, schemas, SQL (ANSI 2011 with extensions), and automatic, synchronous replication for high availability.
 
-- [google-cloud-spanner API documentation](https://googleapis.dev/ruby/google-cloud-spanner/latest)
-- [google-cloud-spanner on RubyGems](https://rubygems.org/gems/google-cloud-spanner)
-- [Google Cloud Spanner API documentation](https://cloud.google.com/spanner/docs)
+- [google-cloud-spanner API
+  documentation](https://googleapis.dev/ruby/google-cloud-spanner/latest)
+- [google-cloud-spanner on
+  RubyGems](https://rubygems.org/gems/google-cloud-spanner)
+- [Google Cloud Spanner API
+  documentation](https://cloud.google.com/spanner/docs)
+
+## NOTICE: Freezing development of `database` and `instance` classes.
+
+From `google-cloud-spanner/v3.0` onwards, **new features for mananging databases
+and instances will only be available through the
+[google-cloud-spanner-admin-instance-v1](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-spanner-admin-instance-v1)
+and
+[google-cloud-spanner-admin-database-v1](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-spanner-admin-database-v1)
+packages**. The `database` and `instance` classes in
```

Comment:
What do you think about a complete listing here of the public methods and types subject to the freeze? This way users could perform a precise audit of usages to be replaced with the new clients.

For example:
```
* `Google::Cloud::Spanner::Project#create_instance`
* `Google::Cloud::Spanner::Project#instance_config`
* `Google::Cloud::Spanner::Project#instance_configs`
...
* `Google::Cloud::Spanner::Instance`
* `Google::Cloud::Spanner::Instance::Config`
* `Google::Cloud::Spanner::Instance::Job`
...
```

Reply:
That's a great idea. I added a list to the FAQ.